modules/reactor/src/meshgenerators/CoreMeshGenerator.C

```diff
@@ -606,6 +612,15 @@ CoreMeshGenerator::generate()
   // Must be called to free the ReactorMeshParams mesh
   freeReactorMeshParams();
 
+  // Return default mesh if option to bypass mesh generation is chosen and free all input meshes
+  if (getReactorParam<bool>(RGMB::bypass_meshgen))
+  {
+    for (const auto & mesh_ptr : _mesh_ptrs)
```

Comment:
could you call getMeshes("inputs") here?
Trying to see if we can get rid of the _mesh_ptrs attribute

Reply:
Unfortunately not, I hit this error message in MOOSE:

![image](https://github.com/idaholab/moose/assets/14334316/6470f1c8-7334-4f3e-8478-c2255bd143dd)

Reply:
Looks like that function calls needs to happen in the constructor and freed in generate